schedule_lessons/manage.py

```diff
@@ -11,5 +11,5 @@
             "Couldn't import Django. Are you sure it's installed and "
             "available on your PYTHONPATH environment variable? Did you "
             "forget to activate a virtual environment?"
-        ) from exc
+        ) from exec
```

Comment:
You marked my previous comment as resolved, but you didn't undo this change haha